package.json

```diff
@@ -12,6 +12,9 @@
     "test": "lerna run --concurrency 2 --stream test",
     "test:update": "lerna run --concurrency 2 --stream test:update",
     "lint": "lerna run lint",
+    "bump-peer-deps": "node scripts/bump-peer-deps.js",
+    "refresh-node-modules": "rm -rf node_modules && rm -rf packages/*/node_modules && npm install",
```

Comment:
Would prefer using `rimraf` for platform independence, but do we really need to blow away node_modules for a new release? Couldn't we just run `npm update --save`?